pkgs/top-level/all-packages.nix

```diff
@@ -8237,6 +8237,8 @@ in
 
   svg2tikz = python27Packages.svg2tikz;
 
+  svg-tikz = (callPackage ../tools/typesetting/svg2tikz {}).svg_tikz {};
```

Comment:
This path doesn't appear to exist. :)

Reply:
Oh right, sorry about that.

Thanks Git.